ios/Codeforces Watcher/Application/core/extensions/NSMutableAttributedString+Extension.swift

```diff
@@ -11,12 +11,16 @@ import UIKit
 
 extension NSMutableAttributedString {
     
-    func colorSubstring(color: UIColor, range: NSRange) -> () {
-        self.addAttribute(NSAttributedString.Key.foregroundColor, value: color, range: range)
+    func colored(with color: UIColor) -> () {
```

Comment:
To make it more flexible, you still can have `range: NSRange? = nil` argument, but with default value of `let range = range ?? NSRange(location: 0, length: self.length)`.